docs/howto/debugging.md

```diff
@@ -158,6 +159,32 @@ call to `createLogger` in `src/boot/middleware.js`.
   doc](https://github.com/evgenyrodionov/redux-logger#options).
 
 
+<div id="immutable-devtools" />
+
+## immutable-devtools: inspect Immutable.js objects in Chrome
+
+Some of the data in the Redux state is stored in Immutable.js data
+structures. It's normally awkward to inspect these data structures; an
+`Immutable.Map` object, for example, is full of properties like
+`size`, `__altered`, `__hash`, and `__ownerID`, which you have to dig
+around to get at a clear representation of the items contained.
+
+[`immutable-devtools`](https://github.com/andrewdavey/immutable-devtools)
+fixes this problem in Google Chrome v47+ by installing a "custom
+formatter".
+
+(Alternatively, we might have recommended a [Chrome
+extension](https://github.com/mattzeunert/immutable-object-formatter-extension),
+which `immutable-devtools` mentions in its setup doc. But using the
+NPM package provides the formatter for `zulip-mobile` just as
+effectively without making widespread changes in behavior when you
+browse the Web.)
+
+To enable it, open Chrome Dev Tools and press F1 to open the settings.
+In the "Console" section, tick "Enable custom formatters". If it isn't
+working, please consult the project's issue tracker before opening an
+issue in `zulip-mobile`.
+
 <div id="webview" />
```

Comment:
nit: double blank line between sections (helps with visually parsing the source)